File: scala/fizzbuzz/src/main/scala/FizzBuzz.scala
case class FizzBuzz() {
  def fizzbuzz(i: Int): String = {
    val fizzWhenMultipleOfThree = if (i % 3 == 0) "Fizz" else ""
    val buzzWhenMultipleOfFive = if (i % 5 == 0) "Buzz" else ""

    if ((fizzWhenMultipleOfThree + buzzWhenMultipleOfFive) != "") fizzWhenMultipleOfThree + buzzWhenMultipleOfFive
    else i.toString
  }
}

File: python/fizzbuzz/fizz_buzz.py
class FizzBuzz:
    def fizzbuzz(self, number):
        res = ""
        if number % 3 == 0:
            res += "Fizz"
        if number % 5 == 0:
            res += "Buzz"

        if res == "":
            res = str(number)
        return res
